### PR TITLE
Remove flow from files

### LINKED ISSUE #9305: Remove Flow from front end
AC:

Flow is removed from FE

### PR DESCRIPTION
**Pull request by youngfreezyVA**
Resolves #9305

### Description
Removes Flow from the frontend

### Acceptance Criteria
- [ ]no regression issues introduced by removing flow

### CODE REVIEW

**Comment by youngfreezyVA** [comment]
@aroltsch in order to get the lint task to pass the circle ci command that runs `lint.rake` needs to be updated to match this: https://github.com/department-of-veterans-affairs/caseflow/commit/7ca8e49e0dcd0adfc50beb3d8ab1de6de1518f17

**Comment by mdbenjam** [comment]
Did you have to do this by hand? Or was there a script that removes flow?

**Comment by youngfreezyVA** [comment]
I did this by hand and re-ran the unit tests in between every few commits. If there was a script I wish I knew about

Get Outlook for iOS<https://aka.ms/o0ukef>

________________________________

File: spec/feature/queue/ama_queue_spec.rb [hunk 1]
```diff
@@ -692,114 +692,6 @@ def valid_document_id
         )
       end
 
-      step "judge sees the case in their review queue" do
```
Comment (mdbenjam) [inline spec/feature/queue/ama_queue_spec.rb]: Why do we need to remove these tests?

File: spec/factories/task.rb [hunk 2]
```diff
@@ -112,6 +112,11 @@
       parent { create(:ama_judge_task) }
     end
 
+    factory :ama_attorney_rewrite_task do
```
Comment (mdbenjam) [inline spec/factories/task.rb]: Why do we need to add this factory?

**Comment by mdbenjam** [comment]
I thought I remember Evan saying there was a removal script when he added Flow, but I'm not actually sure what script that is 🤷‍♂️ . Added two comments, but besides that this looks good to me.

File: spec/feature/queue/ama_queue_spec.rb (hunk 1 above)
Comment (youngfreezyVA) [reply spec/feature/queue/ama_queue_spec.rb]: I branched off of another branch (that is now merged into master) so not sure why this is showing up in the diff, will resolve

**Pull request merged**